README.md

```diff
@@ -456,6 +464,8 @@ Needless to say, engaging the brake while the drum was spinning would be _bad id
 
 ####  008C Ever physically destroy equipment from software?
 
+It may sound funny now, but back in the days hardware wasn't as smart and it was possible to physically damage certain CRT displays by switching them into unusual modes, for example.
```

Comment:
See also question 00A5.

Reply:
Added a reference. It would be nice to properly cross-reference the whole thing...